<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1282</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="qwe">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>大标题</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>左队</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>左类别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>9</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>左赔率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>573</x>
     <y>9</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>比分</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>9</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>右赔率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>856</x>
     <y>9</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>右队</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>997</x>
     <y>9</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>右类别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1138</x>
     <y>9</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>胜者</string>
   </property>
  </widget>
  <widget class="QListWidget" name="Title">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>27</y>
     <width>135</width>
     <height>497</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="LTeam">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>135</width>
     <height>497</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="LType">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>135</width>
     <height>497</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="LOdd">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>27</y>
     <width>135</width>
     <height>497</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="Score">
   <property name="geometry">
    <rect>
     <x>573</x>
     <y>27</y>
     <width>136</width>
     <height>497</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="ROdd">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>27</y>
     <width>135</width>
     <height>497</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="RTeam">
   <property name="geometry">
    <rect>
     <x>856</x>
     <y>27</y>
     <width>135</width>
     <height>497</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="RType">
   <property name="geometry">
    <rect>
     <x>997</x>
     <y>27</y>
     <width>135</width>
     <height>497</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="Winner">
   <property name="geometry">
    <rect>
     <x>1138</x>
     <y>27</y>
     <width>135</width>
     <height>497</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>573</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始抓取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>停止抓取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>读取数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fliteButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>580</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>筛选数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ID3Button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>580</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>ID3训练</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>680</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>680</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>680</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>680</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="YuceAllButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>580</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>决策树预测</string>
   </property>
  </widget>
  <widget class="QPushButton" name="YuceButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>710</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>单独预测</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ANNButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>580</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>ANN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ANNYuceButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>580</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>ANN预测</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>680</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>680</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>680</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>680</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="YuceANNButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>710</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>单独预测</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>550</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>550</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>测试数量</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
